spring-cloud-task-docs/src/main/asciidoc/batch.adoc

```diff
@@ -0,0 +1,102 @@
+
+[[batch]]
+= Batch
+
+[[partintro]]
+--
+This section goes into more detail about Spring Cloud Task's integrations with Spring
+Batch.  Tracking the association between a job execution and the task it was executed
+within as well as remote partitioning via Spring Cloud Deployer are all covered within
+this section.
+--
+
+[[batch-association]]
+== Associating A Job Execution With The Task It Was Executed Within.
```

Comment:
Change to:
Associating A Job Execution  To The Task In Which It Was Executed. (fix on merge)